modules/tinymce/src/plugins/table/main/ts/actions/Clipboard.ts

```diff
@@ -68,6 +68,7 @@ const registerEvents = (editor: Editor, selections: Selections, actions: TableAc
               editor.selection.setRng(data.rng);
               editor.focus();
               cellSelection.clear(table);
+              editor.fire('TableModified', { ...data, table });
```

Comment:
I'm not sure this will work as intended, also `table` is a SugarElement in this context.

Can we use `Events.fireTableModified` like everywhere else?

`import * as Events from '../api/Events';`

```suggestion
              Events.fireTableModified(editor, table.dom, Events.styleAndStructureModified);
```

Reply:
Note: I haven't tested this

Reply:
For additional context, we should pretty much never be calling `editor.fire` from code and it should always be calling a function in the `Events` module.

Reply:
This could also be using `data.effect`, eg:
```suggestion
              Events.fireTableModified(editor, table.dom, data.effect);
```